sdk/openai/azure-ai-openai-realtime/src/main/java/com/azure/ai/openai/realtime/models/RealtimeClientEventConversationItemDelete.java

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+package com.azure.ai.openai.realtime.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Generated;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+
+/**
+ * Send this event when you want to remove any item from the conversation history.
+ */
+@Fluent
+public final class RealtimeClientEventConversationItemDelete extends RealtimeClientEvent {
```

Comment:
Should we use the naming convention of `<EvenType>Event` and drop the `RealTimeClient` prefix? This type would be `ConversationItemDeleteEvent extends RealtimeClientEvent`.

Reply:
I like that and I believe .NET might already be doing something along those lines. I think this is probably best served as a change in the TSP. I will update my PR in that other repo and regenerate.